app/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -20,6 +20,7 @@
 		42127D9B2C283C310005AB40 /* OneSignalExtension.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 42127D9A2C283C310005AB40 /* OneSignalExtension.xcframework */; };
 		42127D9E2C283C400005AB40 /* OneSignalOSCore.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 42127D9D2C283C400005AB40 /* OneSignalOSCore.xcframework */; };
 		42127DA12C283C4E0005AB40 /* OneSignalOutcomes.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 42127DA02C283C4E0005AB40 /* OneSignalOutcomes.xcframework */; };
+		42476A842C5E36DA00426330 /* libsqlite3.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = 42476A832C5E369200426330 /* libsqlite3.tbd */; };
```

Comment:
What are this changes for?

Reply:
This was related to the flutter_downloader package which I removed